db-engine/src/main/java/com/splicemachine/db/iapi/sql/dictionary/MonGetConnectionDescriptor.java

```diff
@@ -0,0 +1,416 @@
+/*
+ * Copyright (c) 2012 - 2020 Splice Machine, Inc.
+ *
+ * This file is part of Splice Machine.
+ * Splice Machine is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU Affero General Public License as published by the Free Software Foundation, either
+ * version 3, or (at your option) any later version.
+ * Splice Machine is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License along with Splice Machine.
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.splicemachine.db.iapi.sql.dictionary;
+
+import org.joda.time.DateTime;
+
+/**
+ * Created by zli on 8/10/20.
+ */
+
+public class MonGetConnectionDescriptor extends TupleDescriptor
+{
+    public static final class MonGetConnectionFields {
```

Comment:
We have to do this because the maximum number of arguments of a method in Java is 255.